### PR TITLE
Added all of my search views and logic.

### PR DESCRIPTION
**Pull request by AAAstorga**
I added all of my views. Including the search bar logic for Monica. Changed all the UIColors to match the extension. Cleaned up all my code. After I merge this, I'll work on the Network.swift file.

### CODE REVIEW

File: TCAT/Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -8,131 +8,156 @@
...
+
+class HomeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, DestinationDelegate {
     
     var tableView : UITableView!
     let userDefaults = UserDefaults.standard
     var searchBar: SearchBarView!
-    let maxNumberOfRecentLocations = 8
+    var recentLocations: [Any] = []
+    
+    let cornellDestinations = [(name: "North Campus", stops: "RPCC, Balch Hall, Appel, Helen Newman, Jessup Field"),(name: "West Campus", stops: "Baker Flagpole, Baker Flagpole (Slopeside)"),(name: "Central Campus", stops: "Statler Hall, Uris Hall, Goldwin Smith Hall"),(name: "Collegetown", stops: "Collegetown Crossing, Schwartz Center"),(name: "Ithaca Commons", stops: "Albany @ Salvation Army, State Street, Lot 32")]
```
Comment (mattbarker016) [inline TCAT/Controllers/HomeViewController.swift]: Could organize the arrays better, use lots of space and new lines

File: TCAT/Cells/RecentSearchCell.swift [hunk 2]
```diff
@@ -15,20 +15,30 @@ class RecentSearchCell: UITableViewCell {
...
-        self.imageView?.contentMode = .scaleAspectFit
-        self.imageView?.center.y = self.bounds.height / 2.0
+        imageView?.frame = CGRect(x: 10, y: 5, width: 20, height: 20)
+        imageView?.contentMode = .scaleAspectFit
+        imageView?.center.y = bounds.height / 2.0
+        imageView?.image = #imageLiteral(resourceName: "search")
+        
+        textLabel?.frame = CGRect(x: 40, y: 0, width: frame.width - 45, height: 20)
+        textLabel?.center.y = bounds.height / 2.0
+        textLabel?.font = .systemFont(ofSize: 13)
```
Comment (mattbarker016) [inline TCAT/Cells/RecentSearchCell.swift]: this spacing, man

File: TCAT/SearchResultsCell.swift [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        
+        imageView?.frame = CGRect(x: 10, y: 5, width: 20, height: 20)
+        imageView?.contentMode = .scaleAspectFit
+        imageView?.center.y = bounds.height / 2.0
+        imageView?.image = #imageLiteral(resourceName: "pin")
+        
+        textLabel?.frame = CGRect(x: 40, y: 5.0, width: frame.width - 45, height: 20)
```
Comment (mattbarker016) [inline TCAT/SearchResultsCell.swift]: should make constants for random numbers, easier to change, describe what they're doing

**Review by mattbarker016 — COMMENTED** [review]
Keep the comments in mind, otherwise, merge on

**Comment by AAAstorga** [comment]
2a0eee8

**Pull request merged**